internal/schema/schema.go

```diff
@@ -54,6 +54,17 @@ type ProjectConfig struct {
 	yaml            *yaml.Node
 }
 
+func NewDefaultProjectConfig() (*ProjectConfig, error) {
```

Comment:
Do we need to return error here? It would be better to simply add a test case for checking if `minimalKlioFile` can be unmarshaled (since it returns always the same data).